app/assets/javascripts/discourse/widgets/post-menu.js.es6

```diff
@@ -61,17 +61,23 @@ function likeCount(attrs) {
         ? "post.has_likes_title_only_you"
         : "post.has_likes_title_you"
       : "post.has_likes_title";
-    const icon = attrs.yours ? "d-liked" : "";
+    let icon = attrs.yours ? "d-liked" : "";
+    let addContainer = attrs.yours;
     const additionalClass = attrs.yours ? "my-likes" : "regular-likes";
 
+    if (!attrs.showLike) {
+      icon = attrs.yours ? "d-liked" : "d-unliked";
```

Comment:
I'm a little confused as to why we only add `d-unliked` when showing likes is disabled?

Reply:
Currently when showing likes is disabled we are displaying just the like count without any icon and that is confusing. With this change we show ❤️ icon next to count to indicate that the count displayed is for likes.

For example, here is how the Like count looked on archived topics for anons before the change:

<img width="79" alt="Screen Shot 2019-06-18 at 22 17 44" src="https://user-images.githubusercontent.com/5732281/59703262-1ffba780-9217-11e9-9d61-1ce8c33bf843.png">

... after the change:

<img width="90" alt="Screen Shot 2019-06-18 at 22 19 00" src="https://user-images.githubusercontent.com/5732281/59703263-1ffba780-9217-11e9-834a-fe2736eb638a.png">

Reply:
Okay I think I get it. It's just a little hard to follow currently. What if you *can* show likes but it's not yours? There's no icon?

Reply:
When like is enabled (`showLike` is true) the like icon is shown via this codepath:

https://github.com/discourse/discourse/blob/7a9e85b1d7a33b2876248b3e2f566606b87a6b00/app/assets/javascripts/discourse/widgets/post-menu.js.es6#L82-L113